src/sgame/sg_bot_nav.cpp

```diff
@@ -586,9 +586,24 @@ static bool BotShouldJump( gentity_t *self, const gentity_t *blocker, const glm:
 	//check if jumping will clear us of entity
 	trap_Trace( &trace, origin, playerMins, playerMaxs, end, self->s.number, MASK_SHOT, 0 );
 
+	if ( blocker->s.modelindex == BA_A_BARRICADE )
+	{
+		glm::vec3 mins, maxs;
+		BG_BoundingBox( static_cast<buildable_t>( blocker->s.modelindex ), &mins, &maxs );
+		if ( G_Team( self ) == TEAM_ALIENS )
+		{
+			maxs.z *= BARRICADE_SHRINKPROP;
+		}
+		glm::vec3 pos = VEC2GLM( blocker->s.origin );
+		BG_BoundingBox( pClass, &playerMins, &playerMaxs, nullptr, nullptr, nullptr );
+		if ( origin.z + playerMins.z >= pos.z + maxs.z )
```

Comment:
This tests whether the player's feet are above the top of the (hypothetically) shrunken barricade. Surely that's not what you want

Reply:
mins are basically the "half-cube" of entities, starting from origin and going downside. Yes, it's not always very readable, I agree. Especially when you think that in fine, the only useful info there is the negative height, as the half-extents can easily be derived from maxs.
But that's not a refactoring I'll do soon for obvious reasons :D

Reply:
yes, they're the only ones allowed to. It's used for *most* models I think. I'm not even sure they're symetrical once.

As I've said on IRC though, I want this as functional as possible before a real refactoring that will be necessary to implement wallwalking.
I hope that having as much features as possible, even if in a PoC state, will make it easier to test for regressions and for reviewers to get what happens. But from my memory, wallwalking is way too fragile to be implemented in such a hackish way as the autojump, crouching or ladder code.